src/main/SeatsioDesigner.js

```diff
@@ -0,0 +1,8 @@
+import AbstractChart from "./AbstractChart";
+
+export default class SeatsioDesigner extends AbstractChart {
```

Comment:
We could rename AbstractChart to Embeddable to make it consistent with the naming in seatsio-core-v2

Reply:
i agree